### ISSUE TITLE
Remove `h1` element around home link

### ISSUE THREAD

**Issue by maxx1128** [maxx1128/eleventy-personal-site#12]
Each page already has an `h1` for the specific title, the consistent home navigation should be a regular link. Check if there are any specific semantics that could/should go with it, like some kind of landmark.
